src/BloomBrowserUI/templates/bloom-foundation-mixins.pug

```diff
@@ -197,6 +197,23 @@ mixin editable(language)
 	//- .bloom-content1.bloom-editable(lang=language, contenteditable="true")&attributes(attributes)
 		block
 
+//- Putting "+clickableLink('href', boolean) here" should result in a clickable link with correct spacing.
+//- Use the boolean argument 'periodAfter' to insert a period directly behind the link.
+//- Putting "+clickableLink('href')#someId here" should give "<a id=someId ..."
+mixin clickableLink(href, periodAfter)
```

Comment:
On the name here: is there a kind of link that isn't clickable? Would link() be just as clear? I'm concerned that camel casing will be so unusual to translators that they'll get it wrong.

"link()" and "link-then-period()" would be easier to read.